frontend/src/components/widgets/CameraInput/WebcamComponent.tsx

```diff
@@ -63,7 +63,7 @@ export const AskForCameraPermission = ({
       <StyledDescription>
         This app would like to use your camera.
         <StyledLink
-          href="https://support.google.com/chrome/answer/2693767"
+          href="https://docs.streamlit.io/knowledge-base/using-streamlit/enable-camera"
```

Comment:
FYI here's a preview of what that page will look like:
https://deploy-preview-237--streamlit-docs.netlify.app/knowledge-base/using-streamlit/enable-camera